framework/include/wifi_manager/wifi_manager.h

```diff
@@ -33,9 +33,13 @@
 
 /* Length defines */
 #define WIFIMGR_MACADDR_LEN        6
-#define WIFIMGR_MACADDR_STR_LEN    18
+#define WIFIMGR_MACADDR_STR_LEN    17
 #define WIFIMGR_SSID_LEN           32
 #define WIFIMGR_PASSPHRASE_LEN     64
+#define MAX_LEVEL_SIGNAL           12
```

Comment:
Defining a define for ST Things in the wifi_manager module was not good.
I do not think you should edit this file.
If it is common in other modules, please raging it to another PR.

Reply:
Previously it was defined in things_types.h file. I will do revert back same changes. and remove 39-42 from here. and define in things_types.h